Lib/imaplib.py

```diff
@@ -1279,6 +1385,236 @@ def print_log(self):
                 n -= 1
 
 
+class _Idler:
```

Comment:
Per my suggestion above, this should be `Idler` for documentation purposes (you'll have to refactor other parts of this PR as well):
```suggestion
class Idler:
```

Reply:
Done in 94c02e88a9d3.